### PR TITLE
Move @graphiql/create-fetcher to @graphiql/toolkit

### PR DESCRIPTION
**Pull request by harshithpabbati**
Moving create fetcher functionalites into toolkit

### CODE REVIEW

**Comment by acao** [comment]
awesome work! can you add to the readme for create fetcher that it was inspired by graphql-subscriptions-fetcher, and thank @urigo?

**Comment by harshithpabbati** [comment]
> @Urigo

**Comment by acao** [comment]
oh hmm, I don't think we're going to be removing the package in this PR. remember we're going to leave it in with the readme pointing to toolkit for one last release?

**Comment by acao** [comment]
i guess we can just delete it though? what do you think @imolorhe or @benjie ? should we just delete the package from npm or leave a waypoint? it has hundreds of downloads since yesterday but that's fine

**Comment by imolorhe** [comment]
If the number of packages using it isn't a lot yet, then it should be fine to delete it? Else i think updating the readme that the functionality has been moved should be fine as well.

**Comment by acao** [comment]
@imolorhe yes, graphiql is only in rc with it. ok sounds good!

wish we’d thought of this before i announced it

thanks so much @harshithpabbati !

**Comment by acao** [comment]
@imolorhe I’ve been meaning to get to this one and its holding up the next RC. if you could take a look that would be awesome!

File: packages/graphiql-toolkit/src/__tests__/buildFetcher.spec.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   createWebsocketsFetcherFromUrl,
   createMultipartFetcher,
   createSimpleFetcher,
-} from '../lib';
+} from '../../dist/lib';
```
Comment (imolorhe) [inline packages/graphiql-toolkit/src/__tests__/buildFetcher.spec.ts:16]: This change should not be necessary

File: packages/graphiql-toolkit/src/__tests__/lib.spec.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { parse } from 'graphql';
-import { isSubscriptionWithName, createWebsocketsFetcherFromUrl } from '../lib';
+import {
+  isSubscriptionWithName,
+  createWebsocketsFetcherFromUrl,
+} from '../../dist/lib';
```
Comment (imolorhe) [inline packages/graphiql-toolkit/src/__tests__/lib.spec.ts:5]: This change should not be necessary.

**Review by imolorhe — APPROVED** [review]

**Pull request merged**